Documentation/contributing.md

```diff
@@ -42,7 +42,7 @@ The submitter has the first responsibility of keeping the created PR clean and n
 A PR is required to meet the following criteria.
 * It has passed all the tests defined for TAOS-CI.
     - This includes unit tests and integration tests in various platforms and different static analysis tools.
-    - Note that one of the tests includes the "Signed-off-by" check, which means that the author has agreed with [Code of Conduct](CODE_OF_CONDUCT.md). You may need to refer to later section.
+    - Note that one of the tests includes the "Signed-off-by" check, which means that the author has agreed with [Code of Conduct](https://github.com/nnstreamer/nnstreamer/blob/main/CODE_OF_CONDUCT.md). You may need to refer to later section.
```

Comment:
How about ```../CODE_OF_CONDUCT.md``` ?

Reply:
This is connected to `nnstreamer.github.io`. So, we may use the complete URL.
- https://nnstreamer.github.io/Documentation/contributing.html